code/modules/reagents/reagent_containers/food.dm

```diff
@@ -120,3 +120,36 @@
 /obj/item/weapon/reagent_containers/food/snacks/tortillachip/New()
 	..()
 	reagents.add_reagent(NUTRIMENT,1)
+
+/obj/poutineocean
+	name = "poutine ocean"
+	desc = "This enormous, horrific cheese wheel is filled to the brim with poutine. You'll probably never reach the bottom."
+	icon = 'icons/obj/food_huge.dmi'
+	icon_state = "poutineocean"
+	pixel_x = -16 * PIXEL_MULTIPLIER
+	pixel_y = -8 * PIXEL_MULTIPLIER
+	var/type_to_dispense = /obj/item/weapon/reagent_containers/food/snacks/poutine
+
+/obj/poutineocean/attack_hand(mob/user)
+	to_chat(user, "<span class='warning'>You need a plate to get food from \the [src]!</span>")
+
+/obj/poutineocean/attackby(obj/item/W, mob/user)
```

Comment:
Might be a good idea to put an
```
else
    ..()
```
in here, if you don't specifically want to exclude the parent procs from being called.